### PR TITLE
Send emails using a default HTML template

### PR DESCRIPTION
**Pull request by danielleknudson**

### CODE REVIEW

**Comment by yefim** [comment]
hmmm I was envisioning something more like:
```` go
// in worker.go

body := *msg.Body
html := RenderTemplate(body, attrs["Template"])

// in templates.go
func RenderTemplate(body, templateName string) string {
  f := os.Open("templates/" + templateName + ".html")
  // render body into the template
  // return the rendered template
}
````

**Comment by danielleknudson** [comment]
Cool, that makes more sense for multiple templates. Should be straightforward to update. Will ping you once it's done.

File: templates.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"bytes"
+	"html/template"
+	"io/ioutil"
+)
+
+type TemplateData struct {
+	Body     string
```
Comment (danielleknudson) [inline templates.go]: A struct will be convenient as we can always add more fields here if the template needs additional data

File: worker.go [hunk 2]
```diff
@@ -120,12 +120,17 @@ func sendEmail(msg *sqs.Message) error {
 		attrs[k] = string(*v.StringValue)
```
Comment (danielleknudson) [inline worker.go]: Once `ReplyTo` is no longer necessary and we know for certain the attributes each message will have, we should remove the `attr` map and access the message attributes via `*msg.MessageAttributes["<AttributeName>"].StringValue`

File: templates.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+		logErr(openErr, nil)
+		return data.Body
+	}
+
+	emailTemplate := template.Must(template.New(data.Template).Parse(string(f)))
+
+	err := emailTemplate.Execute(&markup, data)
+
+	if err != nil {
+		logErr(err, nil)
```
Comment (yefim) [inline templates.go]: we probably want to return `data.Body` in this case

**Comment by yefim** [comment]
LGTM

**Pull request merged**